spec/scrabble-spec.js

```diff
@@ -51,4 +51,91 @@ describe("Scrabble", function() {
 
     expect(scrabble.score()).toEqual(41)
   })
+
```

Comment:
you added lots of extra testing criteria, with some a lot more complicated entries, nice work!